synapse/cores/common.py

```diff
@@ -681,7 +650,48 @@ def initCoreModule(self, ctor, conf):
             logger.warning('mod load fail: %s %s' % (ctor,e))
             return False
 
+    def _synTagsVers0(self):
```

Comment:
I usually prefer having them "close" to where they're used. Is this mostly a question about their order when generating api docs?